client/s3cfg.go

```diff
@@ -35,6 +35,7 @@ var s3cfgTemplates = map[api.CloudProviderType]string{
 	api.Safespring: openstackS3Template,
 	api.CityCloud:  openstackS3Template,
 	api.AWS:        awsS3Template,
+	api.Azure:      exoscaleS3Template, //Azure doesn't support S3 so we'll use exoscale for the POC
```

Comment:
I'd rather skip this whole thing instead. Feels bad merging this to master. Or would it be too much work to make it conditional for Azure?

Reply:
I'd rather have exoscale s3 than nothing. See my comment above as well.

Reply:
Alright, just re-write the comment to say "use exoscale for now" instead of "for the POC" and I'm ok